docs/platform_operators/rotating-secrets-and-certs.md

```diff
@@ -2,7 +2,54 @@
 Most secrets in cf-for-k8s can be rotated by simply changing the values in your `cf-values.yml` file and running a standard deploy using ytt and kapp. The rotation is complete when the kapp deploy succeeds.
 
 ## Exceptions
-As of September 2020, it is currently not possible to rotate `app_registry` credentials. Future work on this will be tracked by https://www.pivotaltracker.com/story/show/173090115
+
+The following fields currently cannot be rotated:
+
+* `blobstore.secret_access_key`
+* `capi.database.encryption_key`
+* `capi.database.password`
+* `cf_admin_password`
+* `cf_db.admin_password`
+* `uaa.database.password`
+
+For example, rotating the cloud controller db encryption key is a breaking change. Rotating the key will 
```

Comment:
Sorry about the late question here given the scope of the story should have been limited to current documentation, but did we happen to validate that the uaa encryption key is rotatable as well?

Reply:
It wasn't in the list of non-rotated keys emitted by `upgrade-cf-new-values` at https://release-integration.ci.cf-app.com/teams/main/pipelines/cf-for-k8s-stability-tests/jobs/validate-values-rotation-redeploy/builds/136

Reply:
Part of the confusion here is that every field has two names: the dotted name from the yaml files, and then the name used internally in the post-ytt yaml. Users will be interacting mostly with the dotted name, but there could be cases where they need to control the actual field name.